static/docs/get-started/example-pipeline.md

```diff
@@ -68,63 +68,62 @@ that are described in earlier [get started](/doc/get-started) chapters.
 > will be determined by the interdependencies between DVC-files, mentioned
 > below.
 
-- Initialize DVC repository (run it inside your Git repository):
+Initialize DVC repository (run it inside your Git repository):
 
-  ```dvc
-  $ dvc init
-  $ git commit -m "initialize DVC"
-  ```
-
-- Download an input data set to the `data` directory and take it under DVC
-  control:
-
-  ```dvc
-  $ mkdir data
-  $ wget -P data https://dvc.org/s3/examples/so/Posts.xml.zip
-  $ dvc add data/Posts.xml.zip
-  ```
+```dvc
+$ dvc init
+$ git commit -m "initialize DVC"
+```
 
-  <details>
+Download an input dataset to the `data` directory and take it under DVC control:
 
-  When we run `dvc add` `Posts.xml.zip`, DVC creates a
-  [DVC-file](/doc/user-guide/dvc-file-format).
+```dvc
+$ mkdir data
+$ wget -P data https://dvc.org/s3/examples/so/Posts.xml.zip
+$ dvc add data/Posts.xml.zip
+```
 
-  ### Expand to learn more about DVC internals
+<details>
```

Comment:
looks like a bug in structure

Reply:
Yeah good find. I only unindented that file, so that error was in before I think, but I just fixed it in f8bbf8f